src/AdoNet/Shared/Storage/AdoNetInvariants.cs

```diff
@@ -21,13 +21,18 @@ internal static class AdoNetInvariants
         /// <summary>
         /// A list of the supported invariants.
         /// </summary>
+        /// <remarks>The invariant names here do not match the namespaces as is often the convention.
+        /// Current exception is MySQL Connector library that uses the same invariant as MySQL compared
+        /// to the official Oracle distribution.</remarks>
         public static ICollection<string> Invariants { get; } = new Collection<string>(new List<string>(new[]
         {
             InvariantNameMySql,
             InvariantNameOracleDatabase,
             InvariantNamePostgreSql,
             InvariantNameSqlLite,
-            InvariantNameSqlServer
+            InvariantNameSqlServer,
+            InvariantNameSqlServerDotnetCore,
```

Comment:
Added here also the SQL Server .NET Core library in order which they appear later. It wasn't catched in the review last time (this is for information purposes if one wants to print all invariants).